src/ci/docker/host-x86_64/x86_64-gnu-llvm-18/Dockerfile

```diff
@@ -51,7 +51,8 @@ ENV RUST_CONFIGURE_ARGS \
       --build=x86_64-unknown-linux-gnu \
       --llvm-root=/usr/lib/llvm-18 \
       --enable-llvm-link-shared \
-      --set rust.thin-lto-import-instr-limit=10
+      --set rust.thin-lto-import-instr-limit=10 \
+      --set rust.lld=false
```

Comment:
Why is this needed? I'd have expected that with these builders we'd not auto-use lld since `llvm-root` is set? Or is that not the same as configuring llvm-config?

Reply:
It is the same and is just for added confidence, I discuss this in the PR text.

Reply:
I've dropped this commit.